### PR TITLE
docs(forms): Custom Validator example selector name incorrect.

### PR DESCRIPTION
**Pull request by MumblesNZ**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #20206 

## What is the new behavior?
Bob error message is now displayed in form validator example. 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: aio/content/examples/form-validation/e2e/app.e2e-spec.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ describe('Form Validation Tests', function () {
     });
 
     tests('Template-Driven Form');
+    bobTests();
```
Comment (mhevery) [inline aio/content/examples/form-validation/e2e/app.e2e-spec.ts]: this does not seem to belong here.

**Review by mhevery — DISMISSED** [review]

**Comment by mhevery** [comment]
also please sign CLA

**Comment by MumblesNZ** [comment]
I've signed it =).

File: aio/content/examples/form-validation/e2e/app.e2e-spec.ts (hunk 1 above)
Comment (MumblesNZ) [reply aio/content/examples/form-validation/e2e/app.e2e-spec.ts]: I would've thought the bob test would apply equally to the template-driven form and reactive form. An e2e test is missing to check for the error that this PR solves, this seemed to fit the bill?

**Comment by MumblesNZ** [comment]
@mhevery could I have an update on this please?

**Comment by tytskyi** [comment]
@MumblesNZ i think you need to write sentence in exact format that bot suggests. Your message did not trigger re-check of the CLA

**Comment by MumblesNZ** [comment]
I signed it!

**Comment by MumblesNZ** [comment]
I signed it!

**Comment by MumblesNZ** [comment]
Thanks @tytskyi, sorted the cla now.

**Comment by alexeagle** [comment]
Ping! We got a duplicate of this PR

**Comment by MumblesNZ** [comment]
Thanks @alexeagle. Just for future reference am I meant to close my own PR? I thought this one was good to go.

**Comment by alexeagle** [comment]
Sorry, that ping was for @mhevery  to move this along.

**Review by kara — APPROVED** [review]
LGTM

**Review by IgorMinar — APPROVED** [review]

**Review by mhevery — APPROVED** [review]

**Pull request closed without merging**